externs/shaka/player.js

```diff
@@ -1459,6 +1459,28 @@ shaka.extern.AdvancedAbrConfiguration;
  */
 shaka.extern.CmcdConfiguration;
 
+
+/**
+ * @typedef {{
+ *   enabled: boolean,
+ *   applyMaximumSuggestedBitrate: boolean,
+ *   estimatedThroughputWeightRatio: number
+ * }}
+ *
+ * @description
+ *   Common Media Server Data (CMSD) configuration.
+ *
+ * @property {boolean} enabled
+ *   If <code>true</code>, enables reading CMSD data in media requests.
+ *   Defaults to <code>true</code>.
+ * @property {boolean} applyMaximumSuggestedBitrate
+ *   Defaults to <code>true</code>.
```

Comment:
This parameter sounds pretty self-explanatory, but it could still use some kind of explanation.

Reply:
Done!